.github/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: General Questions
+    url: https://keptn.sh/community/#slack
+    about: Want to chat? Got a general question? Join us on Slack.
```

Comment:
[nitpick:] please add the missing newline :)

Reply:
```suggestion
    about: Want to chat? Got a general question? Join us on Slack.

```